packages/components/src/border-box-control/types.ts

```diff
@@ -14,6 +19,15 @@ export type AnyBorder = Border | Borders | undefined;
 export type BorderProp = keyof Border;
 export type BorderSide = keyof Borders;
 
+// TODO: replace with Popover actual props once the component
+// gets converted to TypeScript
+export type PopoverPartialProps = {
+	placement?: Placement;
+	offset?: number;
+	anchor?: ReferenceType;
+	__unstableShift?: boolean;
+};
```

Comment:
This should be removed and replace with the original `PopoverProps` type once we type the component (hopefully soon!)

Reply:
Working on it https://github.com/WordPress/gutenberg/pull/43823